ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -2,8 +2,11 @@ presubmits:
   openshift/release:
   - agent: kubernetes
     always_run: false
+    cluster: ci/api-build01-ci-devcluster-openshift-com:6443
     context: ci/openshift/promrules
     decorate: true
+    labels:
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
```

Comment:
`ci/openshift/promrules` is not rehearsed.

Reply:
https://coreos.slack.com/archives/GB7NB0CUC/p1582926172321200?thread_ts=1582921878.312900&cid=GB7NB0CUC